Gemfile

```diff
@@ -3,18 +3,24 @@ source 'https://rubygems.org'
 # This Jekyll version may not be what GitHub Pages supports.
 # If you must ensure the same behaviour as GitHub pages,
 # change this to match what it supports: https://pages.github.com/versions/
-gem 'jekyll', '~>3.8'
+# Uncomment the following line (and comment out the github-pages gem below)
+# to control your own Jekyll version, if not using GH Pages.
+# gem 'jekyll', '~>3.9'
 
-# GitHub Pages recommends using the github-pages gem instead,
-# but this injects unwanted files: https://github.com/github/pages-gem/issues/482
+# GitHub Pages recommends using the github-pages gem instead.
+# This caused issues in the past: https://github.com/github/pages-gem/issues/482
 # Uncomment the following line (and comment out the Jekyll version above)
```

Comment:
Since the default behaviour is changing here, I think this line should read "**Comment out** the following line (and **uncomment** the Jekyll version above)" or something to that effect.

Reply:
@LouiseSteward Ah, nice catch, thanks. I've fixed up these code comments in c31e046.